clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/model/discovery/EurekaInstance.groovy

```diff
@@ -97,7 +99,8 @@ class DiscoveryInstance implements Health {
       vipAddress,
       secureVipAddress,
       lastUpdatedTimestamp,
-      asgName)
+      asgName,
+      metadata?.titusTaskId)
```

Comment:
will do this at a later PR since Jackson doesn't make it easy and don't want to waste too much time on this